roles/setup/tasks/main.yml

```diff
@@ -21,8 +21,8 @@
     mode: '0644'
     owner: "{{ receptor_user }}"
     group: "{{ receptor_group }}"
-  # notify:
-  #   - "Receptor Reload"
+  notify:
+    - Restart Receptor service
```

Comment:
We don't want to be just unconditionally doing a full restart.  The Installer has 3 handlers: a full restart, a reload, and one that catches a failure of the reload and dispatches to restart instead.